File: beginner/__2000__/2540/2540.php
<?php 

    // a corrigir
    while(true){
    
        $input = readline();

        if($input == false || $input == "") break;

        $jogadores = (int) $input;
        $doistercos = ceil($jogadores * (2/3));
        $votos = explode(" ", readline());
        $votosImpeachment = 0;

        for($i = 0; $i < $jogadores; $i++){
            if($votos[$i] == "1"){
                $votosImpeachment++;
            }
        }

        $response = $votosImpeachment >= $doisTercos 
        ? "impeachment" 
        : "acusacao arquivada";

        echo "$response\n";
    }
?>

File: beginner/__2000__/2540/2540.py
while True:
    try:
        jogadores = int(input())
        doisTercos = jogadores * (2 / 3)

        votos = [int(x) for x in input().split()]
        votosImpeachment = 0

        for i in range(jogadores):
            if votos[i] == 1:
                votosImpeachment += 1

        print("impeachment" if votosImpeachment >= doisTercos else "acusacao arquivada")

    except EOFError:
        break